File: api/home-box-landing/HomeBoxLanding.Api/Features/WebSockets/Types/WebSocketKey.cs
namespace HomeBoxLanding.Api.Features.WebSockets.Types;

public enum WebSocketKey
{
    Unknown,
    Handshake,
    BuildStarted,
    BuildUpdated,
    DeployStarted,
    DeployUpdated,
    ServerStats,
    PlexActivity,
    PiholeActivity,
    RadarrActivity,
    SonarrActivity,
    DockerAppUpdateProgress
}

File: src/services/websocket-service/types/web-socket.key.ts
export enum WebSocketKey {
    Unknown = 'Unknown',
    Handshake = 'Handshake',
    BuildStarted = 'BuildStarted',
    BuildUpdated = 'BuildUpdated',
    DeployStarted = 'DeployStarted',
    DeployUpdated = 'DeployUpdated',
    ServerStats = 'ServerStats',
    PlexActivity = 'PlexActivity',
    PiHoleActivity = 'PiHoleActivity',
    RadarrActivity = 'RadarrActivity',
    SonarrActivity = 'SonarrActivity',
    DockerAppUpdateProgress = 'DockerAppUpdateProgress',
}